Makefile.dos

```diff
@@ -0,0 +1,30 @@
+# Makefile for MS-DOS using Watcom compiler.
+#
+# wmake -f Makefile.dos
+# - builds the static library xmp_static.lib
+#
+# To disable module depacker functionality:
+#	wmake -f Makefile.dos USE_DEPACKERS=0
+#
+# To disable ProWizard functionality:
+#	wmake -f Makefile.dos USE_PROWIZARD=0
+#
+!ifndef target
+target = static
+!endif
+
+USE_PROWIZARD	= 1
+USE_DEPACKERS	= 1
+
+CC = wcc386
+SYSTEM = dos4g
+
+CFLAGS = -zq -bt=dos -bm -fp5 -fpi87 -mf -oeatxh -w4 -ei -zp8
```

Comment:
We don't need `-bm` here:
`-bm           build target is a multi-thread environment`

Reply:
We can also use `dos32a` here. Actually, we don't really need to specify anything here (because we build static lib only), but whatever.